django-stubs/contrib/gis/sitemaps/views.pyi

```diff
@@ -1,6 +1,15 @@
-from typing import Any, Optional
+from typing import Optional
+
+from django.http import HttpRequest, HttpResponse
 
 def kml(
-    request: Any, label: Any, model: Any, field_name: Optional[Any] = ..., compress: bool = ..., using: Any = ...
-) -> Any: ...
-def kmz(request: Any, label: Any, model: Any, field_name: Optional[Any] = ..., using: Any = ...) -> Any: ...
+    request: HttpRequest,
+    label: str,
+    model: str,
+    field_name: Optional[str] = ...,
+    compress: bool = ...,
+    using: str = ...,
+) -> HttpResponse: ...
+def kmz(
+    request: HttpRequest, label: str, model: str, field_name: Optional[str] = ..., using: str = ...
+) -> HttpResponse: ...
```

Comment:
These two views are poorly documented in Django itself: https://docs.djangoproject.com/en/4.1/ref/contrib/gis/sitemaps/

I looked in the source to find types: https://github.com/django/django/blob/2bc47d7fe95ec9fc62bc5c0d2408944e5269d9d0/django/contrib/gis/sitemaps/views.py#L10

And at the related URL patterns, which seem only used in Django’s test suite: https://github.com/django/django/blob/2bc47d7fe95ec9fc62bc5c0d2408944e5269d9d0/tests/gis_tests/geoapp/urls.py#L17-L28